cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
for purposes of reading facility and clean code we usually add an aditional line at the end of every file, please add it here

Reply:
Sorry this file is autogenerated and i didn't realize it

Reply:
I decided to delete this file, it is not important because it is a example